### PR TITLE
Fix music fading being completely broken

### PR DESCRIPTION
**Pull request by Crystalwarrior**
~~Reverts AttorneyOnline/AO2-Client#416~~
Closes https://github.com/AttorneyOnline/AO2-Client/issues/431

### CODE REVIEW

**Comment by oldmud0** [comment]
Oh, I know why that broke. Because the FADE_OUT flag is not at the 0th bit position. Duh.

File: src/aomusicplayer.cpp [hunk 1]
```diff
@@ -96,7 +96,7 @@ void AOMusicPlayer::play(QString p_song, int channel, bool loop,
       BASS_ChannelLock(oldstream, false);
     }
 
-    if (effect_flags & FADE_OUT & (m_volume[channel] != 0)) {
+    if (effect_flags & FADE_OUT) {
```
Comment (oldmud0) [inline src/aomusicplayer.cpp:99]:
```suggestion
    if ((effect_flags & FADE_OUT) && m_volume[channel] > 0) {
```

**Review by oldmud0 — APPROVED** [review]
This should fix it

**Comment by Crystalwarrior** [comment]
> 
> 
> This should fix it

Got it

**Comment by Crystalwarrior** [comment]
Fix confirmed to be working, @oldmud0 waiting on merge

**Review by oldmud0 — APPROVED** [review]
Cool and good

**Pull request merged**